src/color/ColorUtils.ts

```diff
@@ -0,0 +1,22 @@
+import * as d3 from "d3";
```

Comment:
I don't know if it would make a difference here, but maybe only import the single function that is used?

Reply:
I resolved this in the most recent commit :)